packages/features/webhooks/components/WebhookForm.tsx

```diff
@@ -40,6 +40,7 @@ const WEBHOOK_TRIGGER_EVENTS_GROUPED_BY_APP_V2: Record<string, WebhookTriggerEve
     { value: WebhookTriggerEvents.BOOKING_PAID, label: "booking_paid" },
     { value: WebhookTriggerEvents.MEETING_ENDED, label: "meeting_ended" },
     { value: WebhookTriggerEvents.RECORDING_READY, label: "recording_ready" },
+    { value: WebhookTriggerEvents.INSTANT_MEETING, label: "instant_meeting_created" },
```

Comment:
I don't think we want that in core, this would also add it as a new trigger event to `settings/developer/webhooks`

Reply:
done